torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -198,24 +199,28 @@ def affine_image_pil(
     return _FP.affine(img, matrix, interpolation=pil_modes_mapping[interpolation], fill=fill)
 
 
-def affine_bounding_box(
+def _affine_bounding_box_xyxy(
```

Comment:
Nit: Why add the implementation in a private method? As far as I understand, rotate could have used the public affine directly. Am I missing something?

Reply:
It is a refactoring of `affine_bounding_box` and this way I also avoid to expose `expand` argument that is not present for `affine_bounding_box`.

Reply:
Thanks for the clarification! So because `affine_image_tensor` doesn't expose an expand, you propose not to expose it on `affine_bounding_box`; makes sense. Should we be exposing expand to affine as an optional parameter or you think that's unnecessary?

Reply:
I know that albu has `expand` as arg for `affine` transformation. I do not know if there is a strong requirement for that. Another point is that we have to code that for `affine_image_tensor` as there is no such option currently. We can create an RFC for that and see community feedback ?

Reply:
Sounds good. Let's create an RFC and move on. We can always revise and add it later.